lib/puppet/network/http.rb

```diff
@@ -5,6 +5,9 @@ module Puppet::Network::HTTP
   MASTER_URL_PREFIX = "/puppet"
   CA_URL_PREFIX = "/puppet-ca"
 
+  PUPPET_API_VERSIONS = "v3, v2"
```

Comment:
Ah, I see. I knew v2 was getting removed but wasn't sure if that was going to be removed for Puppet 4 or not. I've fixed it to only say v3.